doc/changes/devel.rst

```diff
@@ -62,6 +62,7 @@ Bugs
 - Correctly prune channel-specific :class:`~mne.Annotations` when creating :class:`~mne.Epochs` without the channel(s) included in the channel specific annotations (:gh:`12010` by `Mathieu Scheltienne`_)
 - Fix :func:`~mne.viz.plot_volume_source_estimates` with :class:`~mne.VolSourceEstimate` which include a list of vertices (:gh:`12025` by `Mathieu Scheltienne`_)
 - Correctly handle passing ``"eyegaze"`` or ``"pupil"`` to :meth:`mne.io.Raw.pick` (:gh:`12019` by `Scott Huberty`_)
+- Fix bug with :func:`mne.channels.read_channel_adj` (:gh:`11608` by :newcontrib:`Ivan Zubarev`)
```

Comment:
traditionally we put `:newcontrib:` entries at the top, so this should move to somewhere above line 45

Reply:
Also this should mention what was fixed, and the GitHub reference should be to the pull request, not to the issue that got fixed